clusters/app.ci/prow/02_config/_plugins.yaml

```diff
@@ -1863,6 +1879,28 @@ plugins:
   - wip
   - yuks
   - approve
+  integr8ly/application-monitoring-operator:
+  - assign
+  - cat
+  - dog
+  - heart
+  - golint
+  - help
+  - hold
+  - label
+  - lgtm
+  - lifecycle
+  - override
+  - retitle
+  - shrug
+  - sigmention
+  - size
```

Comment:
We recently disabled `size` everywhere because of GH API token consumption, please remove it.

Reply:
Ok, no problem